src/main/java/core/basesyntax/DataProcessing.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public interface DataProcessing {

```

Comment:
[Don't keep all logic in a single package](https://github.com/mate-academy/jv-fruit-shop/blob/master/checklist.md#dont-keep-all-logic-in-a-single-package)